aliquotmaf/annotators/gnomad.py

```diff
@@ -40,24 +40,35 @@ def empty_gnomad_record():
 
 
 class GnomAD(Annotator):
-    def __init__(self, scheme, refpath, refpattern):
+    def __init__(self, scheme, ref_prefix):
         super().__init__(name="GnomAD", scheme=scheme)
-        self.path = refpath
-        self.file_template = refpattern
+        self.file_template = ref_prefix + "{}.feather"
```

Comment:
Ah, this can't join cause join will add in a `/` right? This is a file prefix like `/some/path/gnomad.` which will get changed to `/some/path/gnomad.{}.feather` and the chromosome IDs will be used in the formatted string.

Reply:
yeah the prefix isn't necessarily just a directory path, could have directory path and constant part of file name, or just constant part of file name if it's in working dir

Reply:
I see, I saw this code and thought "Ah, building a path using `+` is a no-no".

I know this class isn't being used right now, but it could use some comments on the expected format(s) of this string.

Reply:
there is annotation in setup describing this btw